components/atom/input/src/inputs/TagInput/index.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import AtomTag from '@schibstedspain/sui-atom-tag'
+import InputWrapper from '../../InputWrapper'
+
+const SEPARATION_KEY = ','
+
+import './styles.scss'
+
+class TagInput extends React.Component {
+  state = {
+    tags: [],
+    inputValue: ''
+  }
+
+  get value () {
+    return this.state.tags.join(SEPARATION_KEY)
+  }
+
+  shouldConvertIntoTag (inputValue) {
+    return inputValue.length > 1 && inputValue[inputValue.length - 1] === SEPARATION_KEY
+  }
+
+  convertIntoTag (inputValue) {
+    const key = inputValue.replace(/,\s*$/, '')
+    this.setState({
+      tags: [...this.state.tags, key],
+      inputValue: ''
+    })
+  }
+
+  removeTag (idx) {
+    this.setState({
+      tags: [
+        ...this.state.tags.slice(0, idx),
+        ...this.state.tags.slice(idx + 1, this.state.tags.length)
+      ]
+    }, this.notifyChange)
+  }
+
+  onChange = (ev) => {
+    const inputValue = ev.target.value
+    if (this.shouldConvertIntoTag(inputValue)) {
+      this.convertIntoTag(inputValue)
+      return
+    }
+
+    this.setState({inputValue}, this.notifyChange)
+  }
+
+  notifyChange = () => {
+    this.props.onChange && this.props.onChange({value: this.value})
+  }
+
+  render () {
+    const {name, label} = this.props
+    return (
+      <InputWrapper name={name} label={label} >
+        {
+          this.state.tags.map((label, idx) =>
+            <AtomTag label={label} key={idx} onClose={() => this.removeTag(idx)} />
```

Comment:
I see in the paper nothing specifies if you can remove tag by backspace, select them, create them with ENTER or something :-(

Reply:
There is this line: "To see how Tags behave inside an Input, please check Autosuggest Component." with a link to https://paper.dropbox.com/doc/SUI-Autosuggest-VHYPgfOTsq4pt50EdimCL. In here there is a tag type in which it says that tags are generated by commas. To remove one Tag, you'll have to click on the tag delete icon